arches_lingo/src/arches_lingo/components/concept/ConceptHeader.vue

```diff
@@ -0,0 +1,179 @@
+    componentName: string;
+    graphSlug: string;
+    resourceInstanceId: string;
+    nodegroupAlias: string;
+}>();
+
+const data = ref<ConceptHeader>();
+
+watch(
+    () => concept.value,
+    (newValue) => {
+        const aliased_data = newValue?.aliased_data;
+
+        const name = newValue?.name;
+        const descriptor = extractDescriptors(newValue, systemLanguage);
+        // TODO: get human-readable user name from resource endpoint
+        const principalUser = "Anonymous"; //newValue?.principalUser; // returns userid int
+        // TODO: get human-readable life cycle state from resource endpoint
+        const lifeCycleState = $gettext("Draft");
+        const uri = aliased_data?.uri?.aliased_data?.uri_content?.url;
+        const partOfScheme =
+            aliased_data?.part_of_scheme?.aliased_data?.part_of_scheme;
+        const parentConcepts = (
+            aliased_data?.classification_status || []
+        ).flatMap(
+            (tile: ConceptClassificationStatusAliases) =>
+                tile?.aliased_data
+                    ?.classification_status_ascribed_classification || [],
+        );
+
+        data.value = {
+            name: name,
+            descriptor: descriptor,
+            uri: uri,
+            principalUser: principalUser,
+            lifeCycleState: lifeCycleState,
+            partOfScheme: partOfScheme,
+            parentConcepts: parentConcepts,
+        };
+    },
+);
+
+onMounted(async () => {
+    try {
+        concept.value = await fetchLingoResource(
+            props.graphSlug,
+            props.resourceInstanceId,
+        );
+    } catch (error) {
+        toast.add({
+            severity: ERROR,
+            life: DEFAULT_ERROR_TOAST_LIFE,
+            summary: $gettext("Unable to fetch concept"),
+            detail: error instanceof Error ? error.message : undefined,
+        });
+    }
+});
+</script>
+
+<template>
+    <div class="header-row">
+        <h2>{{ data?.descriptor?.name }} ({{ data?.descriptor?.language }})</h2>
+        <!-- TODO: Life Cycle mgmt functionality goes here -->
+        <div class="header-item">
+            <span class="header-item-label">{{
+                $gettext("Life cycle state:")
+            }}</span>
+            <span>{{ data?.lifeCycleState }}</span>
+        </div>
+    </div>
+    <div class="header-row">
+        <div class="header-item">
+            <span class="header-item-label">{{ $gettext("URI:") }}</span>
+            <Button
+                :label="data?.uri || '--'"
+                variant="link"
+                as="a"
+                :href="data?.uri"
+                target="_blank"
+                rel="noopener"
+                :disabled="!data?.uri"
+            ></Button>
+        </div>
+    </div>
+    <div class="header-row">
+        <!-- TODO: Human-reable conceptid to be displayed here -->
+        <div class="header-item">
+            <span class="header-item-label">{{ $gettext("Scheme: ") }}</span>
```

Comment:
I imagine the spacing at the end should be handled another way, as this could cause problems for translated strings

Reply:
As a matter of fact, gettext trims the whitespace anyways, so this doesn't work - I had already added margin-inline-end to account for this

Reply:
completely fair, I guess I'm worrying about the orderliness of the created .po file:

```
#: arches_lingo/src/arches_lingo/components/concept/ConceptHeader.vue:121
msgid "Scheme: "
msgstr "Scheme: "

```

Thanks for resolving!